lib/measure_repository_service_test_kit/measure_package.rb

```diff
@@ -140,13 +140,32 @@ class MeasurePackage < Inferno::TestGroup
       input :measure_id, title: 'Measure id'
       uses_request :measure_package
       run do
-        assert(related_artifacts_present?(resource))
+        assert(related_artifacts_present?(resource, false))
       end
     end
 
     test do
-      title 'Throws 404 when no Measure on server matches id'
+      optional
```

Comment:
Nitpick: Maybe optional tests should be the last ones in a group? Not sure, but that feels logical to me rather than having an optional test sandwiched between a bunch of other required ones. Thoughts?